keras-retinanet/detection.py

```diff
@@ -71,11 +70,16 @@ def generate_frames():
   success = True
 
   while success:
+    # If a new model is present, excahnge this with the old model.
+    if "new_model.h5" in os.listdir():
+        sp.call("mv new_model.m5 model.m5", shell=True)
```

Comment:
Er dette kald blokerende?

Reply:
Hvad mener du med blokerende? Altså object detectionen fortsætter ikke mens den skifter model.

Reply:
Netop det jeg var bekymret for, super :)